src/bci_build/package/templates/php_description.j2

```diff
@@ -0,0 +1,3 @@
+PHP is a general-purpose scripting language primarily used for server side web
+development. It can be used as all scripting languages directly, embedded in
+HTML files or executed via a server side Apache2 module or CGI scripts.
```

Comment:
```suggestion
PHP is a general-purpose scripting language used primarily for server-side web
development. It can be used directly, embedded in
HTML files, or executed via a server-side Apache2 module or CGI scripts.
```